src/hotspot/share/opto/superword.cpp

```diff
@@ -3615,3 +3034,4 @@ bool SuperWord::same_origin_idx(Node* a, Node* b) const {
 bool SuperWord::same_generation(Node* a, Node* b) const {
   return a != nullptr && b != nullptr && _clone_map.same_gen(a->_idx, b->_idx);
 }
```

Comment:
Just noticed this by coincidence: This method is dead and can be removed. Can probably also be done in the scope of this PR instead of doing an extra RFE just for that.

Reply:
I'll need to look into the CloneMap separately at some point, so I'll do it then. I'm aware of this one ;)
[JDK-8303113](https://bugs.openjdk.org/browse/JDK-8303113): [SuperWord] investigate if enabling _do_vector_loop by default creates speedup
I hope that one day I can remove the whole `CloneMap` and `_do_vector_loop`.